doc_conf/conf.py

```diff
@@ -80,139 +81,65 @@
-
-# Add any paths that contain custom static files (such as style sheets) here,
-# relative to this directory. They are copied after the builtin static files,
-# so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
-
-
-# -- Options for HTMLHelp output ------------------------------------------
-
-# Output file base name for HTML help builder.
-htmlhelp_basename = "hidimstat_doc"
-
-
-# -- Options for LaTeX output ---------------------------------------------
 
-latex_elements = {
-    # The paper size ('letterpaper' or 'a4paper').
-    #
-    # 'papersize': 'letterpaper',
-    # The font size ('10pt', '11pt' or '12pt').
-    #
-    # 'pointsize': '10pt',
-    # Additional stuff for the LaTeX preamble.
-    #
-    # 'preamble': '',
-    # Latex figure (float) alignment
-    #
-    # 'figure_align': 'htbp',
+# Generate the plots for the gallery
+matplotlib.use("agg")
+sphinx_gallery_conf = {
+    "doc_module": "groupmne",
+    "examples_dirs": ["../examples"],
+    "gallery_dirs": ["auto_examples"],
+    "plot_gallery": "True",
+    "thumbnail_size": (160, 112),
+    "min_reported_time": 1.0,
+    "backreferences_dir": os.path.join("generated"),
+    "abort_on_example_error": False,
+    "image_scrapers": ("matplotlib",),
+    "show_memory": True,
+    "doc_module": "hidimstat",
+    "backreferences_dir": "generated",
+    "reference_url": {
+        # The module we locally document (so, hidimstat) uses None
+        "hidimstat": None,
+        # We don't specify the other modules as we use the intershpinx ext.
+        # See https://sphinx-gallery.github.io/stable/configuration.html#link-to-documentation  # noqa
+    },
 }
 
-# Grouping the document tree into LaTeX files. List of tuples
-# (source start file, target name, title,
-#  author, documentclass [howto, manual, or own class]).
-latex_documents = [
-    (
-        master_doc,
-        "hidimstat.tex",
-        "hidimstat Documentation",
-        "Jerome-Alexis Chevalier",
-        "manual",
-    ),
-]
-
+git_root_url = "https://github.com/mind-inria/hidimstat"
 
-# -- Options for manual page output ---------------------------------------
-
-# One entry per manual page. List of tuples
-# (source start file, name, description, authors, manual section).
-man_pages = [(master_doc, "hidimstat", "Hidimstat Documentation", [author], 1)]
-
-
-# -- Options for Texinfo output -------------------------------------------
-
-# Grouping the document tree into Texinfo files. List of tuples
-# (source start file, target name, title, author,
-#  dir menu entry, description, category)
-texinfo_documents = [
-    (
-        master_doc,
-        "hidimstat",
-        "hidimstat Documentation",
-        author,
-        "hidimstat",
-        "One line description of project.",
-        "Miscellaneous",
-    ),
-]
+html_context = {
+    "display_github": True,  # Integrate GitHub
+    "github_repo": "hidimstat",
+}
```

Comment:
These lines should be move in the option for HTML.

Reply:
done